testsuite/features/support/namespaces/configchannel.rb

```diff
@@ -3,18 +3,49 @@
 
 # "configchannel" namespace
 class NamespaceConfigchannel
+  ##
+  # It initializes the api_test variable.
+  #
+  # Args:
+  #   api_test: This is the test object that is passed in from the test script.
   def initialize(api_test)
     @test = api_test
   end
 
+  ##
+  # > Returns true if the channel exists, false otherwise
+  #
+  # Args:
+  #   channel: The channel to check for existence.
   def channel_exists(channel)
     @test.call('configchannel.channelExists', sessionKey: @test.token, label: channel)
   end
 
+  ##
+  # `list_files` lists the files in a channel
+  #
+  # Args:
+  #   channel: The channel to list files from.
   def list_files(channel)
     @test.call('configchannel.listFiles', sessionKey: @test.token, label: channel)
   end
 
+  ##
+  # "Returns a list of systems subscribed to the given channel."
```

Comment:
Same as above. Why using `""` here?

Reply:
I have the feeling I clicked twice to generate the doc, and the second time it generated doc of the doc: :trollface: